interfaces/builtin/audio_playback.go

```diff
@@ -78,6 +80,12 @@ owner /{,var/}run/user/*/pulse/native rwk,
 owner /{,var/}run/user/*/pulse/pid r,
 `
 
+const audioPlaybackConnectedPlugAppArmorCore = `
+owner /run/user/[0-9]*/###SLOT_SECURITY_TAGS###/pulse/ r,
+owner /run/user/[0-9]*/###SLOT_SECURITY_TAGS###/pulse/native rwk,
+owner /run/user/[0-9]*/###SLOT_SECURITY_TAGS###/pulse/pid r,
+`
```

Comment:
@jhenstridge does this socket has checks about recording or playback, or is it playback only? or is both with no controls yet?

Reply:
If you ask if pipewire-pulse does understand an apply the snap permissions, the answer is: the patch is ready, it has been sent to pipewire and debian-salsa, and there is a PPA with the modified pipewire-pulse, and packages for jammy, karmic, lunar and mantic:

https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/3277

https://salsa.debian.org/utopia-team/pipewire/-/merge_requests/21

https://launchpad.net/~rastersoft-gmail/+archive/ubuntu/pipewire/+packages